exec/java-exec/src/main/resources/rest/index.ftl

```diff
@@ -79,11 +94,15 @@
                   </span>
                 </td>
                 <td id="status" >${drillbit.getState()}</td>
-                <#if (model.shouldShowAdminInfo() || !model.isAuthEnabled()) && drillbit.isCurrent() >
+                <td class="uptime" >Not Available</td>
                   <td>
-                      <button type="button" id="shutdown" onClick="shutdown($(this));"> SHUTDOWN </button>
-                  </td>
+                <#if (model.shouldShowAdminInfo() || !model.isAuthEnabled()) && (drillbit.isCurrent() || !model.isUserEncryptionEnabled()) >
```

Comment:
Please re-visit this condition. `IsUserEncryptionEnabled()` only checks for encryption between JDBC/ODBC client to Drillbit path not for Https. You have to check it using protocol.
You can put the check something like below:
```
if(model.shouldShowAdminInfo() && (drillbit.isCurrent() || (!model.isAuthEnabled() && location.protocol != https))) {
    showShutdownButton();
}
```

Reply:
Fixed